remotecv/utils.py

```diff
@@ -10,10 +10,60 @@
 
 import logging
 
+from redis import Redis, Sentinel
+
 
 class Config:
     pass
 
 
 logger = logging.getLogger("remotecv")  # pylint: disable=invalid-name
 config = Config()  # pylint: disable=invalid-name
+
+
+def redis_client():
+    if config.redis_mode == "single_node":
+        return __redis_single_node_client()
+
+    if config.redis_mode == "sentinel":
+        return __redis_sentinel_client()
+
+    return None
```

Comment:
Nit: maybe it would be a bit better if an exception was raised here instead. Argument parsing enforces one choice or another, but this code isn't anware of such fact. And having `ResultStore`'s `storage` set to `None` (`remotecv/result_store/redis_store.py:14`) can lead to some `AttributeError` being raised for no apparent reason.

Reply:
Changed it to raise a `AttributeError`